docs/interfaces/http.rst

```diff
@@ -503,6 +446,63 @@ Code   Error
 ====== =====================================================================
 
 
+.. _http-bulk-ops:
+
+Bulk operations
+===============
+
+The REST endpoint allows to issue bulk operations which are executed as single
```

Comment:
```suggestion
The HTTP endpoint allows to issue bulk operations which are executed as single
```
Not directly related to your changes, but while we are at it: We refer to the endpoint as "HTTP endpoint" everywhere else on that page, and it is not a REST endpoint. A REST endpoint (or rather REST API) would support the different HTTP methods (GET/POST/PATCH/...) to perform specific actions on a resource identified in the URL.
 
A RESTful HTTP POST would need to look something like `HTTP POST https://cratedb-cluster.com:4200/my_schema/my_table` rather than taking arbitrary SQL on a static endpoint.

Reply:
Thanks. I think @mfussenegger already considered your suggestion within his suggestions, that have just been applied.